backend/server.js

```diff
@@ -0,0 +1,62 @@
+const cors = require("cors");
+const morgan = require("morgan");
+const express = require("express");
+const SequelizeStoreInit = require("connect-session-sequelize");
+const session = require("express-session");
+const Sequelize = require("sequelize");
+const app = express();
+
+app.use(
+  cors({
+    origin: "http://localhost:5173",
+    credentials: true,
+  }),
+);
+app.use(express.json());
+app.use(morgan());
+
+const newSequelize = new Sequelize("event_finder", "postgres", "1025323", {
+  host: "localhost",
+  dialect: "postgres",
+});
+
+const SequelizeStore = SequelizeStoreInit(session.Store);
+const sessionStore = new SequelizeStore({
+  db: newSequelize,
+});
+
+// Session middleware
+app.use(
+  session({
+    secret: "your-secret-key",
```

Comment:
should we be providing an actual secret key?

Reply:
To Do:
Delete sequelize db & pass content to prisma